modules/apps/frontend-token/frontend-token-definition-impl/src/main/java/com/liferay/frontend/token/definition/internal/FrontendTokenDefinitionRegistryImpl.java

```diff
@@ -367,9 +386,9 @@ public void removedBundle(
 	private ClientExtensionEntryRelLocalService
 		_clientExtensionEntryRelLocalService;
 
-	private final Map<String, FrontendTokenDefinitionImpl>
+	private final Map<String, FrontendTokenDefinition>
 		_frontendTokenDefinitionImpls = new ConcurrentHashMap<>();
```

Comment:
The other one is called `_frontendTokenDefinitionsMap`. Its not very clear. Since one Map is related to `theme` and the other with `CET`, what about rename them to `_themeFrontendTokenDefinitionsMap` and `_cetFrontendTokenDefinitionsMap`?

Reply:
I guess that's fine. I'm just no sure if we need the suffix `Map`, maybe we could drop it. If I'm not wrong, we add `Map` when we want to differentiate between a list and a map

```
List<CET> cets = ...;
Map<String, CET> cetsMap = ...
```